content/source/assets/stylesheets/application.scss

```diff
@@ -1,42 +1,42 @@
-@import "bootstrap-sprockets";
-@import "bootstrap";
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
```

Comment:
Can you skip this linting change? Probably we're just out of step with the styles for this repo.